packages/ws/src/utils/constants.ts

```diff
@@ -31,6 +32,17 @@ export const CompressionParameterMap = {
 	[CompressionMethod.ZlibSync]: 'zlib-stream',
 } as const satisfies Record<CompressionMethod, string>;
 
+export const ZlibErrorCodes = {
+	[ZlibSync.Z_NEED_DICT]: 'Z_NEED_DICT',
```

Comment:
Can't import zlib-sync on the top level at runtime. It's being lazy loaded conditionally since it's usage is opt-in
https://github.com/discordjs/discord.js/blob/cf87cbb04103feccb7bd515d89a820adddbe7d8f/packages/ws/src/ws/WebSocketShard.ts#L667

Refer to the parenthesis i included in https://github.com/discordjs/discord.js/pull/10808#discussion_r2071456416